CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## 1.6.2
+* Fix - Resolves fatal error when the plugin can't get the current user [#365](https://github.com/ebanx/woocommerce-gateway-ebanx/pull/365)
```

Comment:
We don't know if the problem is really the current user... Can we change this message?